components/ScheduleSection.js

```diff
@@ -0,0 +1,242 @@
+import SectionTitle from './SectionTitle'
+
+const sessions = [
+  {
+    id: 1,
+    type: 'FLASH',
+    time: '10:00 - 10:10',
+    sessionFirstTopic: 'Welcome Speech by Venue'
+  },
+  {
+    id: 2,
+    type: 'MAIN',
+    time: '10:30 - 11:15',
+    sessionFirstTopic: 'Let\'s start width React in the right way',
+    sessionFirstAvatar: 'https://avatars2.githubusercontent.com/u/25113?v=3&s=400',
```

Comment:
We already have avatar images in our repository. Let’s use that instead.

**Note:** The avatar image file name is based on the speaker’s name. So there is no need to hardcode the avatar URL.